README.rst

```diff
@@ -2,6 +2,11 @@
 prompt_processing
 #################
 
+.. image:: https://github.com/lsst-dm/prompt_processing/actions/workflows/ci-release.yaml/badge.svg?event=push
+   :target: https://github.com/lsst-dm/prompt_processing/actions/workflows/ci-release.yaml
+
+.. Can't think of any useful alt text
```

Comment:
It's unfortunate that the alt text can't be auto generated based on what the badge says.

Perhaps this for alt text: Release CI Build Status Badge

Reply:
I was thinking that that's exactly the sort of thing that isn't helpful in e.g. a screen reader. 😕 Maybe just "Release CI Build Status", which is at least a good name for the hyperlink?